docs/pull_request.md

```diff
@@ -30,6 +30,19 @@ In any case, it is better to explicitly declare in the description why the PR is
 
 Also, draft PR should not stay indefinitely in this state. It may be removed if it is the case and the submitter does not update it after a few days.
 
+##### Base branch
+
+The `develop` branch is generally the base branch for every PRs.
+
+Exceptions can occur:
+
+- if a feature implementation is split into multiple PRs. We can have a chain of PRs in this case. PR can be merged one by one on develop, and GitHub change the target branch to `develop` for the next PR automatically.
+- we want to merge a PR from the community, but there is still work to do, and the PR is not updated by the submitter. In this case, we can create a new branch, push it, and change the target branch of the PR to this new branch. The PR can then be merged, and we can add more commits to fix the issues. After that a new PR can be created with `develop` as a target branch.
+
+**Important notice 1:** Releases are created from the `develop` branch. So `develop` branch should always contain a "releasable" source code. So when a feature is being implemented with several PRs, it has to be disabled by default, until the feature is fully implemented. A last PR to enable the feature can then be created.
```

Comment:
Perhaps we can explicitly state what's meant by a feature being disabled? Like whether it's via feature flag or being kept on another branch, etc

Reply:
When multiple developers are working on the same feature, waiting for others' PR to be merged into develop is sometimes blocking/time-consuming. I think we should consider keeping them in another branch and create new branches on top of it instead of develop.

Reply:
I think working on a different branch for a new feature may be painful concerning resolving conflicts when merging the feature branch into develop: it will raise the same problems as forked project. In my opinion, using a feature flag is the painless approach to smoothly integrate new features. But I admit we should find a good process to be faster to avoid blocks when working on the same feature.

Reply:
> Perhaps we can explicitly state what's meant by a feature being disabled? Like whether it's via feature flag or being kept on another branch, etc

3faf0a238ce29c90f6866128cb97a1158089fa7d